src/components/radioGroup/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useMemo, Fragment, useState } from 'react';
+import styles from './radiogroup.module.scss';
+
+export interface IRadioGroupItem {
+  label: string;
+  value: string;
+}
+
+export interface IProps {
+  onSelect: (value: any) => void;
+  values: IRadioGroupItem[];
+}
+
+/**
+ * Very simple styled radiogroup component that takes an array of radiogroup items and
+ * reports its changes using the given onSelect callback.
+ *
+ * @param props
+ */
+export default function RadioGroup(props: IProps) {
+  const { onSelect, values } = props;
+  const [selected, setSelected] = useState<string>(values?.[0].value);
+
+  const id = useMemo(() => Math.random().toString(36).substr(2), []);
+
+  const onChange = (value: string) => {
```

Comment:
add `: void` return type
```suggestion
  const onChange = (value: string): void => {
```